Ladybird/.github/workflows/ci.yml

```diff
@@ -0,0 +1,131 @@
+on:
```

Comment:
not a fan of keeping these here. Might be fine to drop ladybird CI temporarily if there's more work needed, but let's not keep these files around in a subdi where they won't do anything

Reply:
Removed with `git filter-branch -f --index-filter 'git rm -r --cached --ignore-unmatch Ladybird/.github' master..HEAD`